botCommands/faq.test.js

```diff
@@ -7,6 +7,8 @@ describe('/faq', () => {
     [' /faq'],
     ['/faq @odin-bot'],
     [' /faq @odin-bot'],
+    ['/faq @user'],
```

Comment:
This test case is redundant, the one above it handles the same test case

Reply:
Removed :+1: